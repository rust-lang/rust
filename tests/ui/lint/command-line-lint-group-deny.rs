//@ compile-flags: -D bad-style

fn main() {
    let _InappropriateCamelCasing = true; //~ ERROR should have a snake
}
