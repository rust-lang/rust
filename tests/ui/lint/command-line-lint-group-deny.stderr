error: variable `_InappropriateCamelCasing` should have a snake case name
  --> $DIR/command-line-lint-group-deny.rs:4:9
   |
LL |     let _InappropriateCamelCasing = true;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^ help: convert the identifier to snake case: `_inappropriate_camel_casing`
   |
   = note: `-D non-snake-case` implied by `-D bad-style`
   = help: to override `-D bad-style` add `#[allow(non_snake_case)]`

error: aborting due to 1 previous error

